sdk/core/azure-core/src/main/java/com/azure/core/util/paging/ContinuablePage.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util.paging;
```

Comment:
Not sure if this belongs in `util` - maybe just `com.azure.core.paging`?

Reply:
The poller abstractions are in `util` namespace so thought of being consistent with it, hence used the same package for paging abstractions. @JonathanGiles thoughts?

Reply:
My main line of thinking is that I don't particularly want too many top-level packages. I am personally fine with it under `util`.